custom.Makefile

```diff
@@ -1,2 +1,55 @@
+SPARQLDIR				:=	data/sparql
+
 $(ONTOLOGYDIR)/upheno1-equivalent.owl: $(MIRRORDIR)/upheno1-equivalent.owl
-	robot merge -i $< -o $@
\ No newline at end of file
+	robot merge -i $< -o $@
+
+$(ONTOLOGYDIR)/upheno2-equivalent.owl: $(MIRRORDIR)/upheno2-lattice.owl $(ONTOLOGYDIR)/upheno-mappings-equivalent-class.owl
+	robot query -i $(ONTOLOGYDIR)/upheno-mappings-equivalent-class.owl --update $(SPARQLDIR)/prepare-upheno-mapping.ru \
+	merge -i $< -o $@
+
+
+$(TMP_DATA)/upheno_custom_mapping.sssom.tsv: #is created using phenio-toolkit
+	phenio-toolkit lexical-mapping \
+	--species-lexical $(TMP_DATA)/upheno_species_lexical.csv \
```

Comment:
Where is the goal for $(TMP_DATA)/upheno_species_lexical.csv?

Reply:
Always add _all dependencies_ for all goals into the make goals! This serves as much as documentation as it helps with debugging later.

Reply:
We need to discuss how upheno_species_lexical.csv is generated